test/Sentry.Unity.Tests/Stubs/TestSentryUnity.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Net.Http;
+
+namespace Sentry.Unity.Tests.Stubs
+{
+    public static class TestSentryUnity
+    {
+        internal static IDisposable Init(Action<SentryUnityOptions> configure,
+            Action<HttpRequestMessage>? httpRequestCallback = null)
```

Comment:
nit: Arguably easier to read when args are stacked at the same level:

```suggestion
        internal static IDisposable Init(
            Action<SentryUnityOptions> configure,
            Action<HttpRequestMessage>? httpRequestCallback = null)
```